arbos/programs/programs.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/ethereum/go-ethereum/core/state"
 	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+	gethParams "github.com/ethereum/go-ethereum/params"
```

Comment:
imported same module twice

Reply:
Thanks! I renamed it because we already have params in the function and there'd be a ton of merge conflicts to rename it. I used gethParams in the other place too.